tests/common_test.go

```diff
@@ -41,6 +45,11 @@ func (s *BaseTestSuite) SetupSuite() {
 		panic(fmt.Sprintf("It was unable to connect to the DB.\n%s\n", err))
 	}
 
+	// set all connections will be closed immediately.
+	// this is required to check connections are leaked or not.
+	// because database/sql keep connection in the pool by default.
+	db.SetMaxIdleConns(0)
```

Comment:
You seem to enable this for all tests. Couldn't this be set just for testing idle conns instead of for the whole suite?

Reply:
It is okay be set just for testing idle conns, but I think about feature changes.
In the future, someone write some tests without setting idle conns checking, the test says okay with leaked idle conns.
Then I prefer to be set the whole test suite than to be set just for testing.